src/features/donations/components/PaymentRequestForm.tsx

```diff
@@ -145,12 +146,14 @@ export const PaymentRequestCustomButton = ({
   }, [paymentRequest]);
 
   useEffect(() => {
-    if (paymentRequest) {
+    if (paymentRequest && !paymentLoading) {
```

Comment:
Do you think this change should also be done in https://github.com/Plant-for-the-Planet-org/planet-webapp/blob/develop/src/features/legacyDonations/components/PaymentRequestForm.tsx which is mostly the same code?